hatchways/team-bluejay
# Kareem navbar and logged in User structure #14

Ticket #14 

Description:
- Added material-UI icons, Axios
- Added AuthContext Providor
- Added the reusable ProtectedRoute component
- Created the navbar with a top app bar and a bottom navigation bar
- set up the basic authentication flow  (Will be redirected to to login if there is no logged in user)

Testing:
- NPM install new packages
- navigate to /
- verify that you got redirected to /login
- enter any email and pw
- get redirected to /
- verify that the appbar is as per the specification
- (view in mobile /tablet mode) verify that the bottom navigation is as per the team's agreement
